### DISCUSSION TITLE
Gogs webhooks not working with Jenkins on a TrueNAS server

### DISCUSSION THREAD

**Discussion by Bender-6502** [gogs/gogs#7070] [Q&A]
Both my Jenkins and Gogs are running on separate jails and everything is happy.
However, I just cannot get my webhook setup in Gogs to work, and Gogs never receives a response from Jenkins? Jenkins logs has nothing too.

I have entered Jenkins and Gogs IPs to the LOCAL_NETWORK_ALLOWLIST.

However, the following http post command in Gog's jail shell receives a response:

`curl -H "Content-Type: application/json" -X POST -d '{"user":"bob",
"pass":"123"}' "http://192.168.1.56:8080/gogs-webhook/?job=cpptest"`

`Response: {"result":"ERROR","message":"Only push event can be accepted."}`

Gogs has a log entry after trying the Test Delivery button:

`2022/06/22 23:14:13 [ERROR] [...s.io/gogs/internal/db/webhook.go:490 AfterSet()] Unmarshal [19]: readObjectStart: expect { or n, but found P, error found in #1 byte of ...|Payload URL|..., bigger context ...|Payload URL resolved to a local network address tha|...`

Any ideas?

**Comment by unknwon** [comment]
Thanks! Please follow on https://github.com/gogs/gogs/issues/7074, and the patch to fix this is currently under review.

**Comment by Bender-6502** [comment]
Great newsWhen is the next release?CheersSteveSent from my Galaxy
-------- Original message --------From: Joe Chen ***@***.***> Date: 25/06/2022  12:39  (GMT+00:00) To: gogs/gogs ***@***.***> Cc: SteveE2 ***@***.***>, Author ***@***.***> Subject: Re: [gogs/gogs] Gogs webhooks not working with Jenkins on a TrueNAS server (Discussion #7070) 
Thanks! Please follow on #7074, and the patch to fix this is currently under review.

**Reply by unknwon** [reply]
0.12.10 has just been released: https://github.com/gogs/gogs/releases/tag/v0.12.10
